R/app_server.R

```diff
@@ -4,9 +4,34 @@
 #'     DO NOT REMOVE.
 #' @import shiny
 #' @noRd
+
 app_server <- function(input, output, session) {
-  conn <- callModule(mod_side_panel_server, "side_panel", action)
+  conn <-
+    callModule(
+      mod_dashboard_structure_server,
+      "dashboard_structure",
+      action,
+      action_table_structure,
+      action_query,
+      action_create_table
```

Comment:
Consider using list when more than the server and id arguments are used. So that method signature wont change and backward compatibility is maintained